From: Bruce Rogers <brogers@suse.com>
Date: Mon, 10 Aug 2015 09:15:13 -0600
Subject: kabi: wrapper include file with __GENKSYMS__ check to avoid kabi
	change
Patch-mainline: never
References: bsc920110

Including include/linux/kexec.h in drivers/pci-pcidriver.c caused the
symbols exported from that source file to result in a different kabi
crc in a non-obvious and unintended way. Wrapper that include to avoid
this change.

Signed-off-by: Bruce Rogers <brogers@suse.com>
---
 drivers/pci/pci-driver.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pci/pci-driver.c b/drivers/pci/pci-driver.c
index dd5258a..bbecb5e 100644
--- a/drivers/pci/pci-driver.c
+++ b/drivers/pci/pci-driver.c
@@ -18,7 +18,9 @@
 #include <linux/sched.h>
 #include <linux/cpu.h>
 #include <linux/pm_runtime.h>
+#ifndef __GENKSYMS__
 #include <linux/kexec.h>
+#endif
 #include "pci.h"
 
 struct pci_dynid {

